__tests__/task.test.js

```diff
@@ -1,8 +1,11 @@
 import { internet } from 'faker';
-import { clearDatabaseState, launchApp, shutdownApp } from './helpers.js';
+import {
+  clearDatabaseState, getAuthenticatedUser, launchApp, shutdownApp,
+} from './helpers';
 
```

Comment:
we need to add tests that one user cannot delete other users tasks

Reply:
that not about auth, lets do it in #41  ?

Reply:
separate issue would be better https://github.com/alex-popov-tech/backend-project-lvl4/issues/43